stages/org.osbuild.systemd.unit.create.meta.json

```diff
@@ -142,6 +144,20 @@
                                 "items": {
                                     "type": "string"
                                 }
+                            },
+                            "Environment": {
+                                "type": "array",
+                                "description": "Sets environment variables for executed process.",
+                                "items": {
+                                  "type": "string"
+                                }
+                            },
+                            "EnvironmentFile": {
+                                "type": "array",
+                                "description": "Path to file that contains environment variables to set for the process.",
+                                "items": {
+                                  "type": "string"
+                                }
```

Comment:
Do we want to do this or an array of tuples/array of objects?

Reply:
You mean (key, value) for the env vars?  Probably a good idea.  Should also quote the `"VAR=VAL"` part I guess in case there are spaces https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#Environment

Reply:
I do like the idea but we already have [`Environment` in the drop-in file stage](https://github.com/osbuild/osbuild/blob/6b4bb850a77feade79d839d9ea075e29ca210ae9/stages/org.osbuild.systemd.unit.meta.json#L53-L56) and it's a plain string.  I think it's nicer if it's an object with `key: string, value: string`, but also I don't like being inconsistent.  It's also arguably easier to leave things like quoting for spaces to the user and just pass through the value the user gives us directly.

The inconsistency is my main issue though.  Perhaps we can make the other stage take an object too as an alternative schema for the property.

Reply:
> we can make the other stage take an object too as an alternative schema for the property.

Let's do this.

Reply:
Let's do that :)